index.html

```diff
@@ -31,12 +31,133 @@ <h1 class="about-title">I’m Martin.Glad to see you!</h1>
             <li><a href="#"><img src="image/Vector (13).png" alt="peace"></a></li>
         </ul>
     </section>
+    <!-- WORK SECTION -->
+    <section class="work">
+        <!-- CARD 1 -->
+        <div class="card-1">
+            <img src="image/Snapshoot Portfolio.png" alt="image1" >
+            <h1 class="tonic">Tonic</h1>
+            <div class="canop">
+                <p class="can-info">CANOPY</p>
+                <div class="point"></div>
+                <p class="back">Back End Dev</p>
+                <div class="point"></div>
+                <p class="back">2015</p>
+            </div>
+            <div class="languages">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+            </div>
+            <button class="see">See project</button>
+        </div>
+        <!-- CARD 2 -->
+        <div class="card-2">
+            <img src="image/Snapshoot Portfolio2.png" alt="image1" >
+            <h1 class="tonic-2">Tonic</h1>
+            <div class="canop-2">
+                <p class="can-info-2">CANOPY</p>
+                <div class="point-2"></div>
+                <p class="back-2">Back End Dev</p>
+                <div class="point-2"></div>
+                <p class="back-2">2015</p>
+            </div>
+            <div class="languages-2">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+            </div>
+            <button class="see-2">See project</button>
+        </div>
+        <!-- CARD 3 -->
+        <div class="card-3">
+            <img src="image/Snapshoot Portfolio3.png" alt="image1" >
+            <h1 class="tonic-3">Tonic</h1>
+            <div class="canop-3">
+                <p class="can-info-3">CANOPY</p>
+                <div class="point-3"></div>
+                <p class="back-3">Back End Dev</p>
+                <div class="point-3"></div>
+                <p class="back-3">2015</p>
+            </div>
+            <div class="languages-3">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+            </div>
+            <button class="see-3">See project</button>
+        </div>
+        <!-- CARD 4 -->
+        <div class="card-4">
+            <img src="image/Snapshoot Portfolio4.png" alt="image1" >
+            <h1 class="tonic-4">Tonic</h1>
+            <div class="canop-4">
+                <p class="can-info-4">CANOPY</p>
+                <div class="point-4"></div>
+                <p class="back-4">Back End Dev</p>
+                <div class="point-4"></div>
+                <p class="back-4">2015</p>
+            </div>
+            <div class="languages-4">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+            </div>
+            <button class="see-4">See project</button>
+        </div>
+    </section>
+    <!-- ABOUT SECTION -->
+    <section class="myself">
+        <h1 class="me">About Myself</h1>
```

Comment:
- [ ] Great job on adding the section title! However, there are a few adjustments that need to be made:
  - [ ] It's recommended to use a more appropriate heading tag for the section title. While using `<h1>` is allowed, it's generally recommended to have only one `<h1>` tag per webpage for better accessibility and clarity. Consider using a heading tag that reflects the hierarchy and relevance of the title within your document, such as `<h2>` to `<h6>`. This will ensure a logical heading structure and improve accessibility for users.
  - [ ] Additionally, the spacing around the title doesn't match the design specifications. Please update your code to adjust the spacing and align it with the desired design.
![image](https://github.com/mar4444/re-portfolio/assets/55840999/2c3e04c3-d7e3-4691-9883-eb63fe1e4e1a)

Please make these adjustments to improve the accessibility and visual consistency of your section title. Keep up the good work! 👏🌟